java/HealthVault Mobile/HVSample/src/main/res/layout/weight.xml

```diff
@@ -4,16 +4,28 @@
 	android:layout_height="match_parent">
 
 	<LinearLayout
+		android:id="@+id/graphContainer"
+		android:layout_width="match_parent"
+		android:layout_height="237dp"
+		android:background="#008272">
+		<com.jjoe64.graphview.GraphView
+			android:layout_width="match_parent"
+			android:layout_height="match_parent"
+			android:id="@+id/graph" />
+	</LinearLayout>
+
+	<LinearLayout
```

Comment:
I have a task to complete this:  [https://microsofthealth.visualstudio.com/Health/_workitems/edit/54619](https://microsofthealth.visualstudio.com/Health/_workitems/edit/54619)

---
In reply to: [114015873](https://github.com/Microsoft/healthvault-samples/pull/5#discussion_r114015873) [](ancestors = 114015873)